feature_layers/display-route-layer/src/main/java/com/esri/samples/display_route_layer/DisplayRouteLayerSample.java

```diff
@@ -0,0 +1,168 @@
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // authentication with an API key or named user is required to access basemap and other location services
+            String yourAPIKey = System.getProperty("apiKey");
+            ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+            // create a map with the topographical basemap style
+            ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+            // create a map view and set the map to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // set a viewpoint on the map view
+            mapView.setViewpoint(new Viewpoint(45.2281, -122.8309 + 0.2, 57e4));
+
+            // create a VBox to contain all the UI controls
+            var vBox = new VBox();
+            vBox.setMaxSize(300, 400);
+            vBox.setPadding(new Insets(10.0));
+            vBox.setAlignment(Pos.TOP_CENTER);
+            vBox.setSpacing(10);
+            vBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0, 0, 0 , 0.7)"), CornerRadii.EMPTY, Insets.EMPTY)));
+
+            //create a scrollPane to enable scrolling through the directions
+            var scrollPane = new ScrollPane();
+            scrollPane.setMaxSize(300, 400);
+            scrollPane.setPadding(new Insets(10.0));
+
+            // create a VBox to contain the directions
+            VBox directionsVBox = new VBox();
+            directionsVBox.setSpacing(10);
+            scrollPane.setContent(directionsVBox);
+            var headingLabel = new Label("Directions:");
+            headingLabel.setStyle("-fx-font-weight:bold; -fx-text-fill: white; -fx-font-size: 14;");
+            vBox.getChildren().addAll(headingLabel, scrollPane);
+
+            // create portal and portalItem
+            var portal = new Portal("https://www.arcgis.com/");
+            portalItem = new PortalItem(portal, "0e3c8e86b4544274b45ecb61c9f41336");
+            portalItem.loadAsync();
+            portalItem.addDoneLoadingListener(() -> {
+
+                if (portalItem.getLoadStatus() == LoadStatus.LOADED) {
+                    featureCollection = new FeatureCollection(portalItem);
+                    featureCollection.loadAsync();
+                    featureCollection.addDoneLoadingListener(() -> {
+                        if (featureCollection.getLoadStatus() == LoadStatus.LOADED) {
+                            for (FeatureCollectionTable table : featureCollection.getTables()) {
+                                if (table.getTitle().equals("DirectionPoints")) {
+                                    for (Feature feature : table) {
+                                        String text = (String) feature.getAttributes().get("DisplayText");
+                                        //creating a Label.
+                                        Label label = new Label(text);
```

Comment:
We can use `var` here, and also leave a line break between this line and the comment afterwards.